classifier.go

```diff
@@ -75,31 +75,50 @@ type License struct {
 	// Threshold is the lowest confidence percentage acceptable for the
 	// classifier.
 	Threshold float64
+
+	// db is the path to the license database
+	db string
+}
+
+// OptionFunc set options on a License struct.
+type OptionFunc func(l *License) error
+
+// Database is an OptionFunc to specify the location of the license database.
+func Database(db string) OptionFunc {
```

Comment:
Is Archive a better name than Database here? 

`Database` has connotations of ability to read and write data, but it seems like it's more of a "read only from an archive file".

Reply:
Done.